gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/orc/HiveOrcSerDeManager.java

```diff
@@ -114,13 +112,10 @@ public HiveOrcSerDeManager(State props)
   }
 
   @Override
+  //Unsupported operation since we remove the orc.schema.literal
   public boolean haveSameSchema(HiveRegistrationUnit unit1, HiveRegistrationUnit unit2)
       throws IOException {
-    if (unit1.getSerDeProps().contains(SCHEMA_LITERAL) && unit2.getSerDeProps().contains(SCHEMA_LITERAL)) {
-      return unit1.getSerDeProps().getProp(SCHEMA_LITERAL).equals(unit2.getSerDeProps().getProp(SCHEMA_LITERAL));
-    } else {
-      return false;
-    }
+      throw new UnsupportedOperationException();
```

Comment:
Removing `orc.schema.literal` doesn't mean we give up schema metadata, actually `columns` and `column.types` are still valid and could be used for schema comparison.

Reply:
Will Address